cmd/juju/common/cloud.go

```diff
@@ -157,3 +160,12 @@ type PrintConfigSchema struct {
 	Type        string `yaml:"type,omitempty" json:"type,omitempty"`
 	Description string `yaml:"description,omitempty" json:"description,omitempty"`
 }
+
+func FormatConfigSchema(values interface{}) (string, error) {
```

Comment:
Would be awesome to format this nicer, but seems future work. Yaml is easy to read, but it doesn't fit well with the --help output for controller-config.

Reply:
Yes, but at the moment we are after consistency and this is what model-config does (as well as other output that prints attibutes' help).